test/Microsoft.AspNetCore.SignalR.Tests/ServerFixture.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.PlatformAbstractions;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.SignalR.Tests
+{
+    public class ServerFixture : IDisposable
+    {
+        private readonly bool _verbose;
+
+        private Lazy<string> _baseUrl;
```

Comment:
To delay deployment - if you have some tests in the same test class that are not using the server you don't want spend time on deployment since it is not needed (e.g. when you try to debug this test)